tools/warnings_count/warnings_build_current.txt

```diff
@@ -1,2 +1,3 @@
+[95mWarning: [97mInvalid pointer indirect: 0 for samplebank SampleBank_2[0m
```

Comment:
I'm curious. What produces this warning?

Reply:
This warning comes from the bugged soundfont 40. Soundfont 40 reports a samplebank index of 0 in the soundfont table but it actually uses samplebank 2 (if you look at the sample pointers in the soundfont, they're nonsense when applied to samplebank 0 but make sense for samplebank 2).

Samplebank indices can either refer to a "real" samplebank or a pointer to a real samplebank. I have no idea why this system exists, but unfortunately it does.

The build system treats soundfont 40 as if it is trying to use samplebank 2 through a pointer index of 0. But 0 is not a pointer index, it's a real samplebank, hence the warning.